apps/examples/testcase/le_tc/filesystem/Kconfig

```diff
@@ -16,4 +16,11 @@ config TC_FS_PROCFS
 	default n
 	select FS_PROCFS
 
+config TC_FS_SMART_SFS
+	bool "Secure File System"
+	default n
+	select SMARTFS_SECURE
```

Comment:
I think this will not work. 
```SMARTFS_SECURE``` has many dependancy and it is under conditional.
"depends on" is right I think.

Reply:
Yes, I will change it to depends on.